schema.prisma

```diff
@@ -211,10 +213,31 @@ model Campaign {
   vaults                  Vault[]
   withdrawals             Withdrawal[]
   slugArchive             SlugArchive[]
+  campaignNews            CampaignNews[]
 
   @@map("campaigns")
 }
 
+model CampaignNews {
+  id           String             @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  campaignId   String             @map("campaign_id") @db.Uuid
+  publisherId  String             @map("publisher_id") @db.Uuid
+  slug         String
+  title        String
+  author       String
```

Comment:
The original idea I had was in case an article about the campaign's beneficent was written elsewhere, to be able to easily repost it in our platform as well  giving the full credits to the original article's author. For similar  the `sourceLink` field exists, to track where the article originates from. 

I could make the author field optional on the frontend, and if empty it will use the logged in user's firstname and lastname.

Reply:
> I could make the author field optional on the frontend, and if empty it will use the logged in user's firstname and lastname.

Yes let's do that!